images/github-jira-proxy/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM docker.io/library/golang:1.21 as builder
+WORKDIR /go/src/github.com/kubevirt/project-infra/
+RUN mkdir -p /go/src/kubevirt/ && \
+    cd /go/src/kubevirt/ && \
+    git clone https://github.com/kubevirt/project-infra.git && \
+    cd project-infra/ && \
+    go mod vendor && \
+    env GOPROXY=off GOFLAGS=-mod=vendor CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -o /go/bin/github-jira-proxy ./external-plugins/github-jira-proxy/main.go
```

Comment:
@orenc1 Not sure how to update this to be my repo: https://github.com/avlitman/GitHub-Jira-Auth.

Reply:
if you build the go binary from your repo, i think you should clone that repo in line 5

Reply:
Yes I will upload my image to quay instead of docker-hub, and try to run it in this project.
where is the place I should use my image?